edx_django_utils/monitoring/internal/code_owner/utils.py

```diff
@@ -103,6 +109,58 @@ def get_code_owner_mappings():
     return _PATH_TO_CODE_OWNER_MAPPINGS
 
 
+def _get_catch_all_code_owner():
+    """
+    If the catch-all module "*" is configured, return the code_owner.
+
+    Returns:
+        (str): code_owner or None if no catch-all configured.
+
+    """
+    try:
+        code_owner = get_code_owner_from_module('*')
+        return code_owner
+    except Exception:  # pylint: disable=broad-except; #pragma: no cover
```

Comment:
I think we should really avoid this pattern. At the very least, we should be logging an exception:

```
...
except Exception as e:
    logger.exception(e)
    return None
```

But we should really be more specific about the type of exception that we are catching. In my experience, the only times where we need to catch Exception is when we want to convert to another type of exception. E.g:

```
except Exception as e:
    raise CodeOwnerException from e
```

Reply:
This topic was already discussed on the last PR. I do not plan on making changes at this time. I don’t want to add risk to what I am trying to get done, and this cleanup could be handled separately.

Reply:
FYI: I added temporary custom attributes to all of the broad exceptions.  I imagine that these will simply be deleted, but I am being conservative. I will follow-up in another pass later.